examples/minimal/packages/contracts/mud.config.ts

```diff
@@ -22,6 +23,14 @@ export default mudConfig({
       },
       ephemeral: true,
     },
+    Inventory: {
+      primaryKeys: {
+        user: "address",
+        item: "bytes32",
+        variant: "uint32",
```

Comment:
what is variant?

Reply:
just a random example of a `uint32` key tuple (though of a variant of some item, like "red cup" vs "green cup", lemme know if you can think of a better example haha)

Reply:
ahh, in that case, `itemVariant` might be a little more descriptive